controls/os_spec.rb

```diff
@@ -38,13 +59,19 @@
     it { should exist }
     it { should be_file }
     it { should be_owned_by 'root' }
-    its('group') { should eq 'root' }
     it { should_not be_executable }
     it { should be_writable.by('owner') }
     it { should be_readable.by('owner') }
-    it { should_not be_readable.by('group') }
     it { should_not be_readable.by('other') }
   end
+  describe file('/etc/shadow'), :if => os.family == 'redhat' do
```

Comment:
InSpec does not support the Serverspec/RSpec if conditions `:if => os.family == 'redhat'`. Instead I would prefer to set a variable before and validate the value eg.
```
if os.redhat?
    shadow_group = 'root'
else if os.debian?
    shadow_group = 'shadow'
end
```